pycaret/tests/test_time_series.py

```diff
@@ -611,3 +609,32 @@ def test_tune_model_raises(load_data):
         exceptionmsg
         == f"`search_algorithm` must be one of 'None, random, grid'. You passed '{search_algorithm}'."
     )
+
+
+def test_ts_tests(load_data):
+    """Tests ts_tests module"""
+
+    exp = TimeSeriesExperiment()
+    data = load_data  # _check_data_for_prophet(name, load_data)
+
+    exp.setup(
+        data=data,
+        verbose=False,
+    )
+    result = exp.ts_tests(show_hidden_keys=True)
+    # print(result)
+    assert type(result) == pd.DataFrame
+    assert list(result.index) == [
+        ('Temporal', 'Trend'), ('Temporal', 'Trend Slope'), ('Temporal', 'Seasonal'),
+        ('Temporal', 'Seasonal Period'), ('Correlation', 'Significant ACF lags'),
+        ('Correlation', 'Significant PACF lags'), ('Correlation', 'Highest ACF lag'),
+        ('Correlation', 'Highest PACF lag'), ('Correlation', '_ACF values'),
+        ('Correlation', '_PACF values'), ('Statistics', 'Mean'), ('Statistics', 'Median'),
+        ('Statistics', 'Standard Deviation'), ('Statistics', 'Variance'),
+        ('Statistics', 'Kurtosis'), ('Statistics', 'Skewness'),
+        ('Statistics', 'Count of distinct values'), ('Statistics', '_Data distribution'),
+        ('Statistics', 'Length'), ('Statistics', 'Shapiro normality test'),
+        ('Statistics', '_Normality pvalue'), ('Stationarity', 'ADF stationarity'),
+        ('Stationarity', '_ADF pvalue'), ('Stationarity', 'KPSS stationarity'),
+        ('Stationarity', '_KPSS pvalue'), ('Stationarity', 'Stationarity')
+    ]
```

Comment:
what happens if the values from the result are all the same, can we test the values from the ouput of `exp.ts_tests` ?